style.css

```diff
@@ -0,0 +1,98 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: grid;
+  width: 100%;
+  height: 100%;
+  background: #fff;
+}
+
+body {
+  display: block;
+  background: #fff;
+}
+
+.main-header {
+  height: 50px;
+}
+
+.menu {
+  position: absolute;
+  left: 80%;
+  top: 4%;
+  font-size: 30px;
+}
+
+.main-header .menu {
+  line-height: 4;
+  display: flex;
+}
+
+.part1 {
+  background: url(background.jpeg) gainsboro;
+  background-position: cover;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 10px;
```

Comment:
- [ ] As mentioned in the previous review, Kindly remove the `margin: 10px` to follow the Figma design as requested.
### Your design :point_down: 
![Screenshot from 2023-05-29 13-29-52](https://github.com/mumbereh/my-portfolio/assets/91142432/b2ccd34f-5fc4-499d-bf16-402b40ccf2e0)
### what it is supposed to look like :point_down: 
![241700423-0cff3ad8-e1a0-4c32-b92c-a9b2b8a76a0d](https://github.com/mumbereh/my-portfolio/assets/91142432/3f4f0ec6-95f1-4b0c-b31b-e545d6882547)